.github/workflows/build.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
 
+      - name: Build
+        run: pnpm nx run-many --target=build --parallel
```

Comment:
pnpm nx --help
 WARN  Unsupported engine: wanted: {"node":"18.x"} (current: {"node":"v16.17.1","pnpm":"8.1.1"})
Smart, Fast and Extensible Build System

Commands:
  nx generate <generator> [_..]                    Generate or update source code (e.g., nx generate @nrwl/js:lib mylib). [aliases: g]
  nx run [project][:target][:configuration] [_..]  Run a target for a project
                                                   (e.g., nx run myapp:serve:production).

                                                   You can also use the infix notation to run a target:
                                                   (e.g., nx serve myapp --configuration=production)

                                                   You can skip the use of Nx cache by using the --skip-nx-cache option.
  nx run-many                                      Run target for multiple listed projects
  nx affected                                      Run target for affected projects
  nx affected:apps                                 Print applications affected by changes
                      [deprecated: Use `nx print-affected --type=app --select=projects` instead. This command will be removed in v15.]
  nx affected:libs                                 Print libraries affected by changes
                      [deprecated: Use `nx print-affected --type=lib --select=projects` instead. This command will be removed in v15.]
  nx affected:graph                                Graph dependencies affected by changes                [aliases: affected:dep-graph]
  nx print-affected                                Prints information about the projects and targets affected by changes
  nx daemon                                        Prints information about the Nx Daemon process or starts a daemon process
  nx graph                                         Graph dependencies within workspace                            [aliases: dep-graph]
  nx format:check                                  Check for un-formatted files
  nx format:write                                  Overwrite un-formatted files                                      [aliases: format]
  nx workspace-lint [files..]                      Lint nx specific workspace files (nx.json, workspace.json)
  nx workspace-generator [name]                    Runs a workspace generator from the tools/generators directory
                                                                                                        [aliases: workspace-schematic]
  nx migrate [packageAndVersion]                   Creates a migrations file or runs migrations from the migrations file.
                                                   - Migrate packages and create migrations.json (e.g., nx migrate
                                                   @nrwl/workspace@latest)
                                                   - Run migrations (e.g., nx migrate --run-migrations=migrations.json). Use flag
                                                   --if-exists to run migrations only if the migrations file exists.
  nx report                                        Reports useful version numbers to copy into the Nx issue template
  nx init                                          Adds nx.json file and installs nx if not installed already
  nx list [plugin]                                 Lists installed plugins, capabilities of installed plugins and other available
                                                   plugins.
  nx reset                                         Clears all the cached Nx artifacts and metadata about the workspace and shuts down
                                                   the Nx Daemon.                                               [aliases: clear-cache]
  nx connect                                       Connect workspace to Nx Cloud                        [aliases: connect-to-nx-cloud]
  nx repair                                        Repair any configuration that is no longer supported by Nx.
  nx view-logs                                     Enables you to view and interact with the logs via the advanced analytic UI from Nx
                                                   Cloud to help you debug your issue. To do this, Nx needs to connect your workspace
                                                   to Nx Cloud and upload the most recent run details. Only the metrics are uploaded,
                                                   not the artefacts.
  nx exec                                          Executes any command as if it was a target on the project
  nx watch                                         Watch for changes within projects, and execute commands
  nx show <object>                                 Show information about the workspace (e.g., list of projects)

Options:
  --help     Show help                                                                                                       [boolean]
  --version  Show version number